README.md

```diff
@@ -68,16 +68,16 @@ easy_install --upgrade "ibm-vpc>=0.7.0"
 ```
 
 ## Using the SDK
-For general SDK usage information, please see [this link](https://github.com/IBM/ibm-cloud-sdk-common/blob/master/README.md)
+For general SDK usage information, see the [IBM Cloud SDK Common README](https://github.com/IBM/ibm-cloud-sdk-common/blob/master/README.md).
 
 ## Setting up VPC service
 ```python
 from ibm_vpc import VpcV1
 from ibm_cloud_sdk_core.authenticators import IAMAuthenticator
 from ibm_cloud_sdk_core import ApiException
 
-authenticator = IAMAuthenticator("YOUR_IBMCLOUD_API_KEY")
-service = VpcV1('2020-06-02', authenticator=authenticator)
+authenticator = IAMAuthenticator('apikey')
+service = VpcV1('2021-07-13', authenticator=authenticator)
```

Comment:
We can change this example to `service = VpcV1(authenticator=authenticator)`

Reply:
Done.